packages/config/config/devices/0x0249/psr07.json

```diff
@@ -0,0 +1,85 @@
+// WeBeHome CPH PSR07
+// Smart Color Button
+{
+	"manufacturer": "WeBeHome",
+	"manufacturerId": "0x0249",
+	"label": "PSR07",
+	"description": "Smart Color Button",
+	"devices": [
+		{
+			"productType": "0x0002",
+			"productId": "0x0001",
+			"zwaveAllianceId": [2029, 2049]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Off Level",
+			"description": "",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Basic Set On Level",
+			"description": "",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "Auto Report Battery Time",
```

Comment:
unit?

Reply:
There's no unit reference in the manual nor on the alliance page but the default is 12.  Based on that, I'm going with hours.

Reply:
Actually, looks like the default is 6 hours, per another part of the manual, so I'm going with "0.5 hours"